dev-requirements.txt

```diff
@@ -3,6 +3,8 @@ nose-exclude==0.4.1
 mock==1.3.0
 coverage==3.7.1
 tox==2.1.1
+pathlib
```

Comment:
this is a bug in reFILE; in the packaging I suspect (https://github.com/Sean1708/reFILE/issues/1)